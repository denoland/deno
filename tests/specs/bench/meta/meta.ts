console.log("import.meta.main: %s", import.meta.main);
console.log("import.meta.url: %s", import.meta.url);
