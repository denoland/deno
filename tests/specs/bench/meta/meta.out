Check [WILDCARD]/meta.ts
import.meta.main: false
import.meta.url: [WILDCARD]/meta.ts
    CPU | [WILDCARD]
Runtime | Deno [WILDCARD] ([WILDCARD])

[WILDCARD]/meta.ts

benchmark   time/iter (avg)        iter/s      (min â€¦ max)           p75      p99     p995
----------- ----------------------------- --------------------- --------------------------

